krcal/core/selection_functions.py

```diff
@@ -148,18 +148,21 @@ def selection_in_band(z         : np.array,
     zc = shift_to_bin_centers(zbins)
 
     sel_e = in_range(e, *range_e)
-    mean, sigma, chi2, ok = fit_slices_1d_gauss(z[sel_e], e[sel_e], zbins, ebins, min_entries=5e2)
+    mean, sigma, chi2, ok = fit_slices_1d_gauss(z[sel_e], e[sel_e], zbins, ebins, min_entries=1e2)
```

Comment:
Shall we make min_entries a configurable parameter?

Reply:
we actually provide a nmin when creating the map

Reply:
This is something internal that doesn't affect the end user, If there is a sensible value (not saying there is or that this is one) I think we can stick to it. Here we are only looking to select the Kr band which should be fairly stable.